backend/backend.go

```diff
@@ -852,7 +877,10 @@ func (backend *Backend) Start() <-chan interface{} {
 	if backend.arguments.DevMode() {
 		backend.baseManager.Start()
 	}
+
+	unlock := backend.accountsLock.Lock()
 	backend.initPersistedAccounts()
+	unlock()
```

Comment:
but now that configureHistoryExchangeRates requires the accontsLock, this unlock should be at the very end past configureHistory call. I would `defer` it.

Reply:
damnit! fixed :)